omod/src/main/resources/config.xml

```diff
@@ -181,7 +181,7 @@
 		</conditionalResource>
 		<conditionalResource>
 			<path>/lib/reporting-api-1.10.*</path>
-			<openmrsVersion>1.10.6 - 2.2.*</openmrsVersion>
+			<openmrsVersion>1.10.6 - 2.4.*</openmrsVersion>
```

Comment:
Don't you want this to work on platform versions above 2.4?

Reply:
I need to

Reply:
You can replace `2.4.*` with `1.9.*`

Reply:
done